composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "wigman/ajax-swatches",
+  "name": "eporsche/ajax-swatches",
```

Comment:
I guess this is not intentional?

Reply:
Nope - it's not. I did this change since composer was not able to pickup my repository otherwise.I am not sure why this happens. So this is basically a workaround for myself only and should not be in this pull request!